lib/config/validation.js

```diff
@@ -81,6 +82,14 @@ async function validateConfig(config) {
           });
         }
       } else if (val != null) {
+        if (topLevel === true) {
```

Comment:
I don't think this logic is correct. packageRules do not have to be only at the top level.